src/pages/reports/Reports.tsx

```diff
@@ -1,16 +1,108 @@
-import Strings from "../../utils/Strings";
-import LayoutCard from "../layouts/LayoutCard";
-const Reports = () => {
-    return (
-        <LayoutCard
-            title={Strings.reports}
-            isLoading={false}
-            content={
-                <div className="flex flex-col">
-                   
-                </div>
-            } />
-    );
+import React, { useState } from 'react';
+import { Button, Collapse, DatePicker, Form, Table, Row, Col } from 'antd';
+import LayoutCard from '../layouts/LayoutCard';
+import SectionElement from '../components/SectionElement';
+import Strings from '../../utils/Strings';
+import { usePostReportsMutation } from '../../services/reportsService';
+import { handleErrorNotification } from '../../utils/Notifications';
+import { Reports } from '../../data/reports/report';
+const { Panel } = Collapse;
+
+interface Report {
```

Comment:
can u create a separate class for this?

Reply:
I mena to create another file inside the models folder to put this class, we shouldn't add classes inside the tsx files just to have a good architecture